### ISSUE TITLE
Re-enable arm64 Docker Image build in Github Actions

### ISSUE THREAD

**Issue by TheAxelander** [TheAxelander/OpenBudgeteer#131]
At the moment the build for arm64 architecture fails with below errors for some reason

```
#31 122.5 /usr/share/dotnet/sdk/7.0.103/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.targets(292,5): error MSB4063: The "GenerateRuntimeConfigurationFiles" task could not be initialized with its input parameters.  [/src/OpenBudgeteer.Blazor/OpenBudgeteer.Blazor.csproj]
#31 122.5 /usr/share/dotnet/sdk/7.0.103/Microsoft.Common.CurrentVersion.targets(5479,7): error MSB4018: The "ReadLinesFromFile" task failed unexpectedly. [/src/OpenBudgeteer.Blazor/OpenBudgeteer.Blazor.csproj]
#31 122.5 /usr/share/dotnet/sdk/7.0.103/Microsoft.Common.CurrentVersion.targets(5479,7): error MSB4018: System.NullReferenceException: Object reference not set to an instance of an object. [/src/OpenBudgeteer.Blazor/OpenBudgeteer.Blazor.csproj]
#31 122.5 /usr/share/dotnet/sdk/7.0.103/Microsoft.Common.CurrentVersion.targets(5479,7): error MSB4018:    at InvokeStub_ReadLinesFromFile.get_Lines(Object, Object, IntPtr*) [/src/OpenBudgeteer.Blazor/OpenBudgeteer.Blazor.csproj]
#31 122.5 /usr/share/dotnet/sdk/7.0.103/Microsoft.Common.CurrentVersion.targets(5479,7): error MSB4018:    at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr) [/src/OpenBudgeteer.Blazor/OpenBudgeteer.Blazor.csproj]
#31 122.5 /usr/share/dotnet/sdk/7.0.103/Microsoft.Common.CurrentVersion.targets(5479,7): error MSB4028: The "ReadLinesFromFile" task's outputs could not be retrieved from the "Lines" parameter. Object reference not set to an instance of an object. [/src/OpenBudgeteer.Blazor/OpenBudgeteer.Blazor.csproj]
```

**Comment by TheAxelander** [comment]
Potential root cause: https://github.com/dotnet/runtime/issues/13648#issuecomment-1471263053

Can be potentially fixed in future with this solution: https://github.com/dotnet/dotnet-docker/issues/4388#issuecomment-1459038566

Considering manually build and push Docker Image on an arm64 environment

**Comment by TheAxelander** [comment]
Fixed with #198
